frontend/app/views/fragments/joiner/signedInAs.scala.html

```diff
@@ -1,12 +1,18 @@
-@(returnUri: String, inline: Boolean = false)
+@import abtests.CheckoutFlowVariant
 
-@import configuration.Config.idWebAppSignOutThenInUrl
+@(returnUri: String, inline: Boolean = false, switchUser: Boolean = false, flowVariant: CheckoutFlowVariant = CheckoutFlowVariant.A)
```

Comment:
You can remove the `switchUser: Boolean` parameter, and use `idWebAppSignOutThenRegisterUrl()` below only - this template fragment (`fragments.joiner.signedInAs()`) is only used in [4 places in our codebase](https://github.com/guardian/membership-frontend/search?utf8=%E2%9C%93&q=fragments.joiner.signedInAs) - friend-signup, payment A/B, & free-paid upgrade - and all of those are fine to use this new behaviour, returning the user to where they came from once they're signed in!